src/Files.App/ViewModels/UserControls/StatusCenterViewModel.cs

```diff
@@ -1,17 +1,17 @@
 // Copyright (c) 2023 Files Community
 // Licensed under the MIT License. See the LICENSE.
 
+using System.Windows.Input;
+
 namespace Files.App.ViewModels.UserControls
 {
-	public class OngoingTasksViewModel : ObservableObject, IOngoingTasksActions
+	public class StatusCenterViewModel : ObservableObject
 	{
-		public ObservableCollection<StatusBanner> StatusBannersSource { get; private set; } = new ObservableCollection<StatusBanner>();
-
-		private int medianOperationProgressValue = 0;
+		private int _MedianOperationProgressValue = 0;
 		public int MedianOperationProgressValue
```

Comment:
Can you rename it to `AverageOperationProgressValue`? 
In `UpdateMedianProgress()` (which should be renamed as well) we are calculating the average value not the median